buildbot/osuosl/master/config/builders.py

```diff
@@ -670,17 +670,18 @@
     'builddir': "clang-ppc64le-rhel",
     'factory' : TestSuiteBuilder.getTestSuiteBuildFactory(
                     depends_on_projects=["llvm", "clang", "clang-tools-extra",
-                                         "lld", "compiler-rt"],
-                    checks=['check-all', 'check-runtimes'],
+                                         "lld", "libcxx", "libcxxabi",
+                                         "compiler-rt"],
+                    checks=['check-runtimes', 'check-all'],
                     extra_configure_args=[
                         "-DLLVM_ENABLE_ASSERTIONS=On",
                         "-DCMAKE_C_COMPILER=clang",
                         "-DCMAKE_CXX_COMPILER=clang++",
                         "-DCLANG_DEFAULT_LINKER=lld",
                         "-DLLVM_TOOL_GOLD_BUILD=0",
-                        "-DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/usr",
-                        "-DCMAKE_CXX_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/usr",
-                        "-DLLVM_BINUTILS_INCDIR=/usr/include",
```

Comment:
Did we do this because `LLVM_BINUTILS_INCDIR` is used for building the gold plugin and since we don't build it on this bot (`"-DLLVM_TOOL_GOLD_BUILD=0"`), we just remove it?

Reply:
Yes Amy, thank you, that is why we removed it, I will update the description to explain why we update the CMake invocation: 
- We use gcc-toolset-12 for the compiler toolchain to me runtime library requirements for the libcxx testing, some test cases fail without it.
- We remove the specification of the LLVM_BINUTILS_INCDIR as we do not build the gold plugin.
- We add ccache to improve on build times.